cpu/numatop.py

```diff
@@ -40,14 +40,22 @@ def setUp(self):
         '''
 
         # Check for basic utilities
-        # TODO: Add support for other distributions
         self.numa_pid = None
-        detected_distro = distro.detect().name.lower()
-        if detected_distro not in ["ubuntu", 'debian']:
-            self.cancel("Upsupported OS %s" % detected_distro)
+        distro_name = distro.detect().name.lower()
         smm = SoftwareManager()
-        for package in ['gcc', 'numatop', 'make', 'libnuma-dev',
-                        'libncurses-dev', 'check', 'pkg-config']:
+        deps = ['gcc', 'numatop', 'make']
+        if 'ubuntu' in distro_name:
```

Comment:
`distro_name == 'ubuntu'` would be better

Reply:
I did not change the original syntax, but yeah this sounds better. Will update.